testsuite/single_node_performance_values.tsv

```diff
@@ -1,55 +1,54 @@
-no-op	1	VM	5	0.992	1.024	30821.1
-no-op	1000	VM	5	0.993	1.004	29796.1
-apt-fa-transfer	1	VM	5	0.988	1.009	21994.4
-apt-fa-transfer	1	NativeVM	5	0.953	1.081	38925.3
-account-generation	1	VM	5	0.983	1.006	18118.3
-account-generation	1	NativeVM	5	0.895	1.120	39070.2
-account-resource32-b	1	VM	5	0.990	1.023	28717.2
-modify-global-resource	1	VM	5	0.993	1.002	1872.7
-modify-global-resource	100	VM	5	0.991	1.030	25730.9
-publish-package	1	VM	5	0.994	1.010	1117.3
-mix_publish_transfer	1	VM	5	0.989	1.015	17197.3
-batch100-transfer	1	VM	5	0.990	1.019	811.3
-batch100-transfer	1	NativeVM	5	0.787	1.179	1699.9
-vector-picture30k	1	VM	5	0.985	1.023	122.5
-vector-picture30k	100	VM	5	0.876	1.018	2187.7
-smart-table-picture30-k-with200-change	1	VM	5	0.974	1.030	16.7
-smart-table-picture30-k-with200-change	100	VM	5	0.974	1.011	226.8
-modify-global-resource-agg-v2	1	VM	5	0.964	1.011	29520.8
-modify-global-flag-agg-v2	1	VM	5	0.996	1.011	3154.0
-modify-global-bounded-agg-v2	1	VM	5	0.967	1.048	5843.5
-modify-global-milestone-agg-v2	1	VM	5	0.964	1.009	21176.2
-resource-groups-global-write-tag1-kb	1	VM	5	0.987	1.001	7518.8
-resource-groups-global-write-and-read-tag1-kb	1	VM	5	0.996	1.029	4400.8
-resource-groups-sender-write-tag1-kb	1	VM	5	0.915	1.128	19863.3
-resource-groups-sender-multi-change1-kb	1	VM	5	0.972	1.050	16962.9
-token-v1ft-mint-and-transfer	1	VM	5	0.984	1.007	805.5
-token-v1ft-mint-and-transfer	100	VM	5	0.984	1.011	10723.1
-token-v1nft-mint-and-transfer-sequential	1	VM	5	0.992	1.011	555.4
-token-v1nft-mint-and-transfer-sequential	100	VM	5	0.989	1.032	8007.9
-coin-init-and-mint	1	VM	5	0.988	1.035	19462.7
-coin-init-and-mint	100	VM	5	0.995	1.044	14651.7
```

Comment:
everything is mostly small improvements (seem to correlate the most with rust version upgrade on May 7th), except for these two, that were caused by FA flags (#16273)